com.obsproject.Studio.Plugin.VerticalCanvas.metainfo.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.VerticalCanvas</id>
+  <extends>com.obsproject.Studio</extends>
+  <name>Vertical Canvas Plugin</name>
```

Comment:
Please remove the `Plugin` suffix. App stores communicate that this is a plugin by placing it in "Add-ons" section.

(I know other plugins have it, but soon I'll start sending pull requests removing that suffix from them too)

Reply:
I was under the impression it had to be this for the plugin extension? So it can be named anything, as long as it `extends` correctly?

Reply:
Yes. The `name` field is an "arbitrary" user-visible field, it isn't usually parsed for anything, except perhaps some basic UTF-8 validation. What's parsed by machines are the `id` and `extends`, but they're already correct in this manifest.